srcpkgs/ergo/INSTALL.msg

```diff
@@ -0,0 +1 @@
+Oragono was renamed to Ergo since version 2.7.0.
```

Comment:
It's missing a trailing newline

And I think it should be `oragono.INSTALL.msg` so only the people using the old package get the warning.

Reply:
I see, will do in just a minute